mrp_bom_line_net_qty/models/mrp_bom_line.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2023 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class MrpBomLine(models.Model):
+    _inherit = "mrp.bom.line"
+
+    product_qty_net = fields.Float(
+        string="Net quantity",
+        help="Weight after any preparation",
+        digits="Product Unit of Measure",
+    )
+    loss_percentage = fields.Float(
+        string="Loss %",
+        help="Percentage loss, e.g., when grating carrots",
```

Comment:
here too

Reply:
and here → "Percentage loss during process"